server/src/main/java/org/apache/iotdb/db/metadata/MLogWriter.java

```diff
@@ -38,6 +41,8 @@
   private static final String STRING_TYPE = "%s,%s,%s"; 
   private File logFile;
   private BufferedWriter writer;
```

Comment:
We can use file channel to replace it. Check other place where it is used ~

Reply:
Fixed